logger/logger.go

```diff
@@ -616,3 +628,70 @@ func getLabels(data map[string]interface{}) []attribute.KeyValue {
 
 	return attrs
 }
+
+//Log is a basic logging method that fulfills the WagClientLogger interface.
+func (l *Logger) Log(level wcl.LogLevel, title string, m map[string]interface{}) {
+	m["title"] = title
+	l.logWithLevel(LogLevel(level), m)
+}
+
+//NewWagClientLogger creates and returns a concrete logger instead of an interface to use with WagClientLogger.
+//This was created because New() returned a KayveeLogger instead of a concrete type.
+func NewWagClientLogger(title string) *Logger {
```

Comment:
Ah, I see why you want to return the concrete logger. That way you only need to initialize once. Since this is a departure from pattern, I would confirm it's ok first in the team chat. If it's ok I would also just name this `NewConcreteLogger`. 

Can the contents of this method be DRY'd up into a private method then shared with `NewLoggerWithContext`?

Reply:
Yeah, I was trying to support our standard use case from the [Readme.md](https://github.com/Clever/kayvee-go/blob/master/README.md) where a logger is created once and reused. We definitely don't want to be initing a new logger each time we log something as it's a fairly heavy process. 

Will bring it up in standup. This makes the most sense to me, and then yeah, I can make that code into a reusable method that both NewConcreteLogger and NewWithContext use.